build/msw/wx_xrc.vcxproj

```diff
@@ -511,6 +511,7 @@
     <ClCompile Include="..\..\src\xrc\xh_stbox.cpp" />
     <ClCompile Include="..\..\src\xrc\xh_stlin.cpp" />
     <ClCompile Include="..\..\src\xrc\xh_sttxt.cpp" />
+    <ClCompile Include="..\..\src\xrc\xh_styledtextctrl.cpp" />
```

Comment:
This is not the right place to add it to (and it will probably result in CI failures in the DLL build), this file should be in wxSTC library, not here.